Broken sub-package cause error even if it is not depended from testing target

github.com/ToQoz/a/main.go:

```
package main

func main() {
}
```

github.com/ToQoz/a/main_test.go:

```
package main

import (
        "flag"
        "os"

        "github.com/ToQoz/gopwt"
        "github.com/ToQoz/gopwt/assert"

        "testing"
)

func TestMain(m *testing.M) {
        flag.Parse()
        gopwt.Empower()
        os.Exit(m.Run())
}

func TestFoo(t *testing.T) {
        assert.OK(t, 1 == 1)
}
```

github.com/ToQoz/a/broken/main.go:

```
package main
```

```
$ go test
[ERROR] go install flag github.com/ToQoz/gopwt github.com/ToQoz/gopwt/assert os testing ./...

# github.com/ToQoz/a/broken
runtime.main_main·f: relocation target main.main not defined
runtime.main_main·f: undefined: "main.main"

exit status 127
```

--- a/_integrationtest/regression/issue44/broken/main.go
+++ b/_integrationtest/regression/issue44/broken/main.go
@@ -1,0 +1,1 @@
+package main

--- a/_integrationtest/regression/issue44/main.go
+++ b/_integrationtest/regression/issue44/main.go
@@ -1,0 +1,4 @@
+package issue44
+
+func main() {
+}

--- a/_integrationtest/regression/issue44/main_test.go
+++ b/_integrationtest/regression/issue44/main_test.go
@@ -1,0 +1,21 @@
+package issue44
+
+import (
+	"flag"
+	"os"
+
+	"github.com/ToQoz/gopwt"
+	"github.com/ToQoz/gopwt/assert"
+
+	"testing"
+)
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+	gopwt.Empower()
+	os.Exit(m.Run())
+}
+
+func TestFoo(t *testing.T) {
+	assert.OK(t, 1 == 1)
+}

--- a/_integrationtest/test.sh
+++ b/_integrationtest/test.sh
@@ -22,24 +22,23 @@ trap cleanup INT QUIT TERM EXIT
 echo "workspace = $workspace"
 cd "$workspace"
 
-(
-  cd "$workspace/regression/issue33"
-  go test
-  go test
-  go test
-)
+for issue in issue33 issue40 issue44
+do
+  (
+    echo "test $issue"
+    cd "$workspace/regression/$issue"
+    go test
+    go test
+    go test
+  )
+done
 
-(
-  cd "$workspace/regression/issue40"
-  go test
-  go test
-  go test
-)
-
+# test vendoring
 (
   GOPATH=$(dirname $workspace)
   export GOPATH
 
+  echo "test issue36"
   cd "$workspace/regression/issue36"
   dep ensure
   rm -rf "vendor/github.com/ToQoz/gopwt"

--- a/translator/internal/types.go
+++ b/translator/internal/types.go
@@ -44,7 +44,7 @@ func GetTypeInfo(vendor string, hasVendor bool, pkgDir, importPath, tempGoSrcDir
 	}
 
 	if IsBuildableFileSet(fset) {
-		installDeps = append(installDeps, "./...")
+		installDeps = append(installDeps, ".")
 	}
 
 	if len(installDeps) > 0 {

